extraction/visitor.go

```diff
@@ -61,7 +60,7 @@ func (v *visitor) visitTypeSpec(typeSpec *ast.TypeSpec) error {
 		Methods: methods,
 	}
 
-	v.specs[title(typeSpec.Name.Name)] = spec
+	v.specs[typeSpec.Name.Name] = spec
```

Comment:
Removing this everywhere will cause things generated to be like Mockthing instead of MockThing. If you only want it to match the command line params we should do this in a different way.

Was that your only concern?

Reply:
I think the command line should match the casing of the type, yeah.  That was my only concern with that.